packages/js/core/src/manifest/formats/web3api.deploy/deserialize.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable */
+/**
+ * This file was automatically generated by scripts/manifest/deserialize-ts.mustache.
+ * DO NOT MODIFY IT BY HAND. Instead, modify scripts/manifest/deserialize-ts.mustache,
+ * and run node ./scripts/manifest/generateFormatTypes.js to regenerate this file.
+ */
+
+import {
+  DeployManifest,
+  AnyDeployManifest,
+  migrateDeployManifest,
+  validateDeployManifest,
+  latestDeployManifestFormat,
+} from ".";
+import { DeserializeManifestOptions } from "../../";
+
+import { compare } from "semver";
+import YAML from "js-yaml";
+import { Tracer } from "@web3api/tracing-js";
+
+export const deserializeDeployManifest = Tracer.traceFunc(
+  "core: deserializeDeployManifest",
+  (manifest: string, options?: DeserializeManifestOptions): DeployManifest => {
+    let anyDeployManifest: AnyDeployManifest | undefined;
+    try {
```

Comment:
Why don't we load json with yaml loader? Since YAML is superset of JSON, it should be possible.